config/software/datadog-gohai.rb

```diff
@@ -27,6 +27,7 @@
   # Windows depends on the registry, go get that.
   if ohai["platform"] == "windows"
     command "#{gobin} get golang.org/x/sys/windows/registry", :env => env
```

Comment:
you'll need to not compile yet here, so `go get -d` if I recall correctly. And then checkout the right commi

Reply:
Good catch.